src/go/rpk/pkg/cli/cmd/wasm/remove.go

```diff
@@ -0,0 +1,67 @@
+package wasm
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/utils"
+)
+
+func NewRemoveCommand(
+	createProduce func(bool, int32) (sarama.SyncProducer, error),
+) *cobra.Command {
+
+	command := &cobra.Command{
+		Use:	"remove <name>",
+		Short:	"remove inline WASM function",
+		Args:	cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			name := args[0]
+			producer, err := createProduce(false, -1)
+			if err != nil {
+				return err
+			}
+			return remove(
+				name,
+				producer,
+			)
+		},
+	}
+
+	return command
+}
+
+func remove(name string, producer sarama.SyncProducer) error {
+	// create empty message, the remove command doesn't need
+	// information on message, just a key value
+	var emptyMessage []byte
+	header, err := createActionHeader(name)
+	if err != nil {
+		return err
+	}
+	//publish message
+	return kafka.PublishMessage(
+		emptyMessage,
+		name,
+		producer,
+		kafka.CoprocessorTopic,
+		[]sarama.RecordHeader{header},
+	)
+}
+
+func createActionHeader(name string) (sarama.RecordHeader, error) {
+	// create key
+	key, err := utils.WriteString([]byte{}, "action")
```

Comment:
can we modify this too? Lets set action to "remove" just to be extra specific

Reply:
Sorry I couldn't understand, this like create a `key`, the key on this case is `"action"`, and value next line is the value "name" == "remove"

Reply:
ah i see it ok resolving